### PR TITLE
feat(dashboard): add initial feature

### PR DESCRIPTION
**Pull request by dhichii**
This feature has a total of:
- Patient
- Doctor
- Nurse
- Polyclinic
- Queue
## Test
- [x] Show total data

### CODE REVIEW

File: libs/api-spec/api-spec.yml [hunk 1]
```diff
@@ -987,6 +1015,21 @@ components:
...
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/Meta_200OK'
+              - type: object
+                properties:
+                  data:
+                    type: object
+                    properties:
+                      total:
+                        type: string
```
Comment (thisham) [inline libs/api-spec/api-spec.yml:1031]: prefer to change it into `number` or `integer`

**Review by thisham — APPROVED** [review]
Test OK. But, need a little adjusment to the documentation

File: libs/api-spec/api-spec.yml (hunk 1 above)
Comment (dhichii) [reply libs/api-spec/api-spec.yml:1031]: ohhh, my bad

Comment (dhichii) [reply libs/api-spec/api-spec.yml]: done https://github.com/ALTERA-CAPSTON-41/backend/pull/45/commits/d25c353f51a81b4f48af42a4bd511325bf5ffe26

**Pull request merged**